providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -5,6 +5,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.AdditionalAnswers.*;
```

Comment:
please try to avoid star imports

Reply:
i think, this is one of the things spotless is not able to properly address sadly. need to check, maybe we add a checkstyle rule after all for that